spec/models/storage_spec.rb

```diff
@@ -11,7 +11,13 @@ def generate_csv_upload(name)
     )
   end
 
-  subject { build :storage }
+  subject { build :storage, user: user }
+
+  let(:user) { User.first }
+
+  before do
+    create :user, email: 'fred@va.gov', password: 'fuggedabodit'
+  end
```

Comment:
Can you describe to me what's going on here? Even after reading the thoughtbot article in your OP, I don't quite follow.

Reply:
When I updated Factorybot the static attributes in the various factories were causing errors when running rspec.  The article mentions that a cop was created for Rubocop to easily fix this issue, in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/467 Rubocop was updated to gain access to the new cop and auto-correct all factories.

Reply:
For these specific changes I'm not sure why they broke but the issue seems to be with how the object is constructed and references an existing User object on creation, errors for things changed were around the User of the model being nil so I set the user property in the tests that were failing

Reply:
Gotcha, I get it now. J/c, why are you overriding the `email` and `password` attributes over using the `Faker` attrs that the users factory uses?

Reply:
copy paste from another spec, removed